### PR TITLE
Fixes closeLastOpenedWindow

### PR DESCRIPTION
**Pull request by lewishadden**
# Contribution description
Fixes bug in closeLastOpenedWindow where the last window is closed and the browser attempts to switch to that closed window

# Pull request checklist
- [x] Contributed code respects the [editorconfig rules](.editorconfig)
- [x] Contributed code passes the [eslint rules](.eslintrc.yaml)
- [ ] Added rules are described in the [readme file](README.md)

### CODE REVIEW

**Review by christian-bromann — APPROVED** [review]
Thank you 👍

File: src/features/elementPosition.feature [hunk 1]
```diff
@@ -10,7 +10,7 @@ Feature: Test the position of a given element
         When  I scroll to element "#square100x100"
 
     Scenario: The element #square100x100 is at XX pixels on the X axis
-        Then  I expect that element "#square100x100" is positioned at 32.5px on the x axis
+        Then  I expect that element "#square100x100" is positioned at 40px on the x axis
```
Comment (christian-bromann) [inline src/features/elementPosition.feature]: Unfortunately the values are different depending if you run locally or in CI

```suggestion
        Then  I expect that element "#square100x100" is positioned at 32.5px on the x axis
```

**Comment by christian-bromann** [comment]
@lewishadden would you mind sign the CLA?

**Comment by lewishadden** [comment]
> @lewishadden would you mind sign the CLA?

Signed

**Comment by christian-bromann** [comment]
> Signed

The bot didn't pick it up, make sure you sign it with the same email you used to make these commits.

**Comment by lewishadden** [comment]
> > Signed
> 
> The bot didn't pick it up, make sure you sign it with the same email you used to make these commits.

Ah yeah was using the wrong account. Signed now

**Pull request merged**